src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -98,6 +105,38 @@ func loadDomains() {
 	domains = strings.Split(string(domainsBytes), "\n")
 }
 
+// Scans the whitelist directory and stores the domains in the files found
+func loadWhitelist(path string, info os.FileInfo, err error) error {
+	if info.IsDir() {
+		// skip root directory
+		return nil
+	}
+	whitelistBytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Fatalf("Unable to read blacklist file at %s: %s", path, err)
+	}
+	for _, domain := range strings.Split(string(whitelistBytes), "\n") {
+		// skip empty lines, comments, and Iranian domains
```

Comment:
Since we're focusing on Iran with this first release, we decided to skip adding *.ir sites to the global whitelist to avoid proxying sites that are already unblocked there. This is a general problem when you aren't maintaining country-specific whitelists. We'll probably address that in the next phase. I added a comment to make this clearer.